From de114bf378c0f402cd8dc8b4adcbee72a94952aa Mon Sep 17 00:00:00 2001
From: Wentao Guan <guanwentao@uniontech.com>
Date: Wed, 8 May 2024 18:11:37 +0800
Subject: [PATCH 48/67] DEEPIN: net: stmmac: Add phytium old dwmac
 acpi_device_id

As Phytium ACPI Description Specification document v1.2 p13
Device HID said for old v1.0 Spec,add FTGM0001 for such as
some FT2000 device compat.

Signed-off-by: Wentao Guan <guanwentao@uniontech.com>

Link: https://github.com/deepin-community/kernel/commit/5c2ef68bfa7f52128c717efb29c3891152fb1c16
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c b/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
index 6e8e44730bec..078e3ee52bdc 100644
--- a/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
+++ b/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
@@ -198,6 +198,7 @@ MODULE_DEVICE_TABLE(of, phytium_dwmac_of_match);

 #ifdef CONFIG_ACPI
 static const struct acpi_device_id phytium_dwmac_acpi_ids[] = {
+	{ .id = "FTGM0001" }, // compat FT2000/4 id
 	{ .id = "PHYT0004" },
 	{ }
 };
--
2.50.1
